src/clearCompleted.js

```diff
@@ -0,0 +1,21 @@
+import { createList } from './addtask.js';
+
+export default function clearAll(items) {
```

Comment:
- [] Please try to use the newer es6 syntax in situations where it is possible. It is recommended to use the newer arrow function syntax.
something like this would be nice
```suggestion
export default (items) => {
```